lib/fxpay/index.js

```diff
@@ -69,14 +69,13 @@
     }
 
     var partialProdInfo = {productId: productId};
-    var log = settings.log;
 
     if (!onPurchase) {
       onPurchase = function _onPurchase(err, returnedProdInfo) {
         if (err) {
           throw err;
         }
-        console.log('product', returnedProdInfo.productId, 'purchased');
```

Comment:
In my head at the time I thought it would be a good idea to use the real console since this is just a default callback for when you forget to specify one. Using console really doesn't seem to make sense now though. Heh.